src/VisionModel/resnet.jl

```diff
@@ -0,0 +1,91 @@
+using Flux
+
+function basicblock()
+  dilation = 1;
+  if dilation > 1
+    error("Dilation > 1 not supported in Basic Block");
+  end
+  return dilation
+end
+
+function bottleneck(planes::Int, stride::Int)
+  stride = 2
+  base_width = 64;
+  groups = 1;
+  width = Int(planes * (base_width / 64.)) * groups;
+  return width, stride
+end 
+
+function identity(inplanes::Int, planes::Int)
+  inplanes = inplanes;
+  planes = planes;
+  return inplanes,planes
+end
+
+function resnet()
+  planes = 64;
+  inplanes = 64;
+  expansion = 1;
+  stride = 1;
+  layers = Chain(
+      Conv((7, 7), 3=>planes, stride=(2, 2), pad=(3, 3)),
+      BatchNorm(planes, λ=relu),
+      MaxPool((3, 3), stride=(2, 2), pad=(1, 1))
+      );
+    basic =  Chain(
+      Conv((3, 3), inplanes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes, λ = relu),
+      Conv((3, 3), planes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes)
+    );
+  push!(layers, SkipConnection(basic, basicblock());
```

Comment:
I assume `basicblock` is still in progress? Right now it just returns an int.

Reply:
Yes, that will return add blocks to the layers

Reply:
i am rechecking it again and will push it in few minutes.